src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -628,6 +629,169 @@ JSON example:
+| m0scan          | The M0 image is a calibration image, used to estimate the equilibrium magnetization of blood.                                                                                          |
+| deltam          | The deltaM image is a perfusion-weighted image, obtained by the subtraction of `control` - `label`.                                                                                    |
+| cbf             | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197)). |
+
+If the `control` and `label` images are not available, their derivative `deltam` should be stored within the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv` instead. If 
+the `deltam` is not available, `cbf` should be stored within the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`. When `cbf` is stored within the `*_asl.nii[.gz]`, its 
+units need to be specified in the `*_asl.json` as well. See below examples of these cases, in order of decreasing preference.
+
+#### Case 1: `*_asl.nii[.gz]` consists of volume_types `control`, `label`
+
+In most cases, the ASL timeseries, provided by the scanner, consist of a series of `control` and `label`, and optionally `m0scan`, volumes. In this case, only the `control`, 
+`label`, and optionally `m0scan` volumes should be stored in the `*_asl.json`, and the exact volume_type series should be specified in the `*_aslcontext.tsv`. The optional 
+`deltam` or `cbf` volumes should be stored and specified as derivative. Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+control
+label
+control
+label
+m0scan
+```
+
+#### Case 2: `*_asl.nii[.gz]` consists of volume_types `deltam` (scanner does not export `control` or `label` volumes)
+
+In some cases, `control` and `label` volumes are lacking within the acquired ASL timeseries, but the intermediate `deltam` - and optionally a `m0scan` - volume is 
+reconstructed/exported by the scanner. In this case, the `deltam` should be included in the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`. An optional `cbf` volume 
+should be stored and specified as derivative. Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+deltam
+m0scan
+```
+
+#### Case 3: `*_asl.nii[.gz]` consists of volume_type `cbf` (scanner does not export `control`, `label`, or `deltaM` volumes) 
+
+If `control` and `label` or intermediate ASL volumes are not reconstructed or exported, but a pre-calculated `cbf` - and optionally a `m0scan` - volume is provided by the
+scanner, the `cbf` should be included in the `*_asl.nii[.gz]` and specified in the `*_aslcontext.tsv`. Example of `*_aslcontext.tsv`:
+
+```Text
+volume_type
+cbf
+m0scan
+```
+
+### Scaling
+The `*_asl.nii.gz` and `*_m0scan.nii.gz` should contain appropriately scaled data, and no additional scaling factors are allowed other than the scale slope in the respective 
+NIfTI headers.
```

Comment:
I think a bit more context for this statement might help. Here's some text that reflects my understanding:

```suggestion
ASL sequences may record in scale factors (slope and intercept) that affect
the interpretation of the DICOM data and which can potentially vary by time point.
NIfTI defines `scl_slope` and `scl_inter` scale factors that apply to the entire image.
The `*_asl.nii.gz` and `*_m0scan.nii.gz` images should contain appropriately scaled data
when applying NIfTI scale factors.
To avoid ambiguity, additional scale factors should not be defined in JSON sidecar files.
```

Please treat this as a suggestion and feel free to reject or further rework.

Reply:
@effigies Thanks for the suggestion. I am adding @HenkMutsaerts here, as he has more insight in the scaling issues. Henk, what do you think about the suggestion?